app/frontend/wallet/constants.ts

```diff
@@ -27,7 +27,8 @@ export const NETWORKS = {
       name: 'mainnet',
       networkId: 1,
       protocolMagic: 764824073,
-      ttl: 10000000,
+      maxTtl: 10000000,
```

Comment:
What we should do at least, is make this constant bigger. But I think we should find a way to calculate this in case the request fails. We don't want tx to have TTL for years nor we want to update this constant every month. I would probably remove this constant altogether.

In addition to this, I would maybe add time representing the start of Shelley era (since it was the moment from which the epochs have a slot every second) and use this constant to calculate ttl by adding time passed since this moment.

Reply:
Fixed